charts/policy-reporter/Chart.yaml

```diff
@@ -5,7 +5,7 @@ description: |
   It creates Prometheus Metrics and can send rule validation events to different targets like Loki, Elasticsearch, Slack or Discord
 
 type: application
-version: 2.18.3
+version: 2.18.4
```

Comment:
You can let it 2.18.3 the helm lint for versions is not mandatory. I bump the version per release